### ISSUE TITLE
Can I select iw, iw, aw, aW etc?

### ISSUE THREAD

**Issue by effinsky** [SUSTech-data/wildfire.nvim#8]

**Comment by fecet** [comment]
This is planned, but I haven't found a way to handle a word other than calling vim's original `viw`, which makes implementation somewhat troublesome. If you have any better ideas, please let me know.

**Comment by effinsky** [comment]
nope. shame, though. was hoping this would be a much more intelligent and less treesitter-dependent region selection aka intellij.
